src/expr/src/scalar/func/impls/numeric.rs

```diff
@@ -299,6 +299,37 @@ sqlfunc!(
     }
 );
 
+sqlfunc!(
+    #[sqlname = "pg_size_pretty"]
+    #[preserves_uniqueness = false]
+    fn pg_size_pretty(mut a: Numeric) -> Result<String, EvalError> {
+        let mut cx = numeric::cx_datum();
+        let units = ["bytes", "kB", "MB", "GB", "TB", "PB"];
+
+        for (pos, unit) in units[..units.len() - 1].iter().enumerate() {
```

Comment:
Isn't this just `units.iter()`?

Reply:
The `- 1` excludes `PB`, but I had a similar thought when I first read it, so maybe there's a more obvious approach?

Reply:
This is more idiomatic:
```
    let strings = ["a", "b", "c", "d"];
    assert!(["a", "b", "c"].iter().eq(strings.iter().rev().skip(1).rev()));
```

Reply:
Thanks!